app/controllers/green_lanes/applicable_exemptions_controller.rb

```diff
@@ -20,33 +20,29 @@ def create
         next_page = determine_next_page
         redirect_to handle_next_page(next_page)
       else
-        @back_link_path = determine_back_link_path
+        @back_link_path = back_link_path_for_current_page
```

Comment:
don't we need a back_link when the form is not valid?

Reply:
We dont because when the page has an invalid form its still using the new back link and ive tested this works for cat 1 questions and cat 2 questions